### PR TITLE
feat: add cuda to CI (didn't realize it was supported)

### PR DESCRIPTION
**Pull request by cryptonemo**
fix: add blstrs gpu features

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -251,6 +251,16 @@ commands:
 
             ./scripts/build-release.sh filcrypto $(cat ./rust-toolchain) build --verbose --locked --no-default-features --features multicore-sdr,opencl
             ./scripts/package-release.sh $TARBALL_PATH
+      - run:
+          name: Build the optimized release (cuda)
```
Comment (dignifiedquire) [inline .circleci/config.yml:255]: do we not run tests on these?

Comment (cryptonemo) [reply .circleci/config.yml:255]: Apparently not, only default features seem to have the tests run

Comment (cryptonemo) [reply .circleci/config.yml:255]: I'll remove those builds and swap for tests

Comment (cryptonemo) [reply .circleci/config.yml:255]: Failed with no nvcc, as I originally expected. Can try to add cuda via apt I guess

Comment (cryptonemo) [reply .circleci/config.yml]: No nvidia-cuda-toolkit.  What am I missing?

Comment (cryptonemo) [reply .circleci/config.yml]: Ah, I see, the ubuntu image

Comment (cryptonemo) [reply .circleci/config.yml]: No go yet, I've removed everything except for the blstrs update for now.  I'll create an issue for CI testing of CUDA but don't want it to block the release

Comment (vmx) [reply .circleci/config.yml]: On CI you need to make sure that the CUDA stuff (`/usr/local/cuda-11.2/bin`) is in the `PATH`, see e.g. https://github.com/filecoin-project/rust-gpu-tools/blob/d3b36477c7d2017e2b7ea284fd91342a667835db/.circleci/config.yml#L26

**Comment by vmx** [comment]
These days CUDA is enabled by default, hence running on CI.

**Pull request closed without merging**